CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 10.0.0-dev.0
+
+- Add support for ObjC Blocks that can be invoked from any thread, using
+  NativeCallable.listener.
+- Bump min SDK version to 3.2.
```

Comment:
`3.2.0-114.0.dev`

We can modify it to 3.2 once we release 10.0.0. (If we ever want to release a dev version for users it would be nice if the changelog would be precise.)